src/sdk/account/utils/toSigner.ts

```diff
@@ -31,6 +43,33 @@ export async function toSigner({
   signer: UnknownSigner
   address?: Address
 }): Promise<LocalAccount> {
+  // ethers Wallet does not have type property
+  if (!signer.type) {
```

Comment:
Can we find a property unique to ethers for the check here? So we can throw a proper message when an unknown signer is passed in here